src/lib/rustContext.ts

```diff
@@ -169,11 +180,13 @@ export default class RustContext {
   }
 
   // Helper to check if context instance exists
-  private async _checkInstance() {
+  private async _checkInstance(): Promise<Context> {
     if (!this.ctxInstance) {
       // Create the context instance.
       await this.create()
```

Comment:
`create()` also needs to return it, and we return that. `tsc` can't tell that calling `this.create()` sets `this.ctxInstance`.

Reply:
okay pushed